src/constants.js

```diff
@@ -431,3 +431,4 @@ export const GA_EVENT_ACTION = {
   SET_PAGE_SIZE: 'Set page size'
 };
 
+export const ROW_THRESHOLD = 90000;
```

Comment:
This should be called for what it is: DISEASE_GENES_MAX_ROWS or something better like that.

Reply:
This is applicable to all the data tables, would it be better to have more generic name like DATA_TABLE_MAX_ROWS?

Reply:
This should not be for all datatables it should only before this table?

Reply:
Ahh ok... so we at least need to alert Chris that this is going to be a global change and not just for this table, and see if he is good with it. Then I would call it DOWNLOAD_BUTTON_THRESHOLD or something like that.